testing/test_server.go

```diff
@@ -204,10 +207,26 @@ func (TestServer) HalfDuplexCall(str grpc_testing.TestService_HalfDuplexCallServ
 }
 
 const (
-	MetadataReplyHeaders  = "reply-with-headers"
+	// MetadataReplyHeaders is a request header that contains values that will
+	// be echoed back to the client as response headers. The format of the value
+	// is "key: val". To have the server reply with more than one response
+	// header, supply multiple values in request metadata.
+	MetadataReplyHeaders = "reply-with-headers"
```

Comment:
The linter complained about exported symbols that had no Go doc, so I fleshed these out.